client/src/document/on-github.tsx

```diff
@@ -89,11 +90,16 @@ function NewIssueOnGitHubLink({ doc }: { doc: Doc }) {
   const baseURL = "https://github.com/mdn/content/issues/new";
   const sp = new URLSearchParams();
 
+  const browser_compat = () => {
+    throw new Error("I don't know how to get the front matter from here");
+  };
```

Comment:
Welcome to a quick "lesson" in the difference between `doc` and `document` in Yari :)
When we read a `index.(html|md)` from disk in the Yari builder that's the `document`. I belive you can use `document.metadata['browser-compat']`. 
```js
const { Document } = require("@mdn/yari/content");

const document = Document.findByURL(
  "/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach"
);
console.log(document.metadata["browser-compat"]);
```
this will print `javascript.builtins.Array.forEach` (or `undefined` if the URL's index.md hasn't got the front-matter key)

Then in the Yari builder, it builds up a `doc` object. This thing has all the bits and pieces that the React server-side and client-side renderer needs. It's quite similar in structure but deliberately different. It's also serialized to a `index.json` file ([for example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach/index.json)) and it's what's serialized into the DOM ([look for `<script type="application/json" id="hydration">`](view-source:https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)).

The reason not everything is in the `doc` is because of web performance. But also for sustainable maintenance. If we pile in everything and anything but end up never rendering it, ever, we're wasting bytes. 
So if you want to put more things from `document` into `doc`, look around [these lines](https://github.com/mdn/yari/blob/d7a33ed83dd53679af68b67af503e3ffd584d805/build/index.js#L435-L441) for inspiration. 

By the way, I find it a little bit odd that we'd want that value here. All the other values are unequivocally always present and always 1 value. But `browser-compat` is "vague". It's strange because not all documents have it so you'd need a fallback string. And some pages have multiple BCD tables so they might not be using the front-matter, and if they do, how do you make the connection to which BCD table it was that mattered. 

Anyway, I hope this helps to unblock.